src/v2/services/implementations/AssetsService.ts

```diff
@@ -58,6 +59,11 @@ export class AssetsService implements IAssetsService {
     const provider = getEvmProvider(this.currentWallet as any);
     const web3 = new Web3(provider as any);
 
+    const resultCheckProvider = await handleCheckProviderChainId(provider);
+    if (!resultCheckProvider) {
+      throw Error('Please connect to the correct network in your wallet');
```

Comment:
use i18

Reply:
@bobo-k2 This is service file. Are you suggesting me to use the `useI18` from out side of the service file?

Reply:
Actually, I have found a way to use i18 in service and repository file ([ref](https://github.com/AstarNetwork/astar-apps/pull/1233/commits/f84a7e237b1a7c3f4d75e5ccf23a6c506e3b488e))!!

![image](https://github.com/AstarNetwork/astar-apps/assets/92044428/a9333979-901e-4714-90d8-0129f697fe61)